internal/clients/idler.go

```diff
@@ -36,7 +36,7 @@ type clusterView struct {
 // IdlerService provides methods to talk to the idler client
 type IdlerService interface {
 	IsIdle(tenant string, openShiftAPIURL string) (bool, error)
-	UnIdle(tenant string, openShiftAPIURL string) error
+	UnIdle(tenant string, openShiftAPIURL string) (*int, error)
```

Comment:
We should be using an `int` instead of `int * ` here. 
I am not sure how a pointer to a const is even going to work.